src/vm/assembly.cpp

```diff
@@ -2349,49 +2249,6 @@ void DECLSPEC_NORETURN Assembly::ThrowBadImageException(LPCUTF8 pszNameSpace,
 
 
 #ifdef FEATURE_COMINTEROP
```

Comment:
I assume we should also remove the `FEATURE_COMINTEROP` as well?

Reply:
There is more COMINTEROP stuff under this ifdef.